diff -Naur a/plugins/generic/webFeed/templates/atom.tpl b/plugins/generic/webFeed/templates/atom.tpl
--- a/plugins/generic/webFeed/templates/atom.tpl	2015-08-28 18:09:59.889889126 +0200
+++ b/plugins/generic/webFeed/templates/atom.tpl	2015-09-11 23:41:34.520202132 +0200
@@ -78,6 +78,9 @@
 				{/if}
 
 				{* optional elements *}
+                {foreach from=$article->getGalleys() item=galley}
+                    <dai-file>{url page="article" op="download" path=$article->getBestArticleId($currentJournal)}{"/"}{$galley->getId()}</dai-file>
+                {/foreach}{* galleys *}
 				{* <category/> *}
 				{* <contributor/> *}
 
diff -Naur a/plugins/generic/webFeed/templates/rss2.tpl b/plugins/generic/webFeed/templates/rss2.tpl
--- a/plugins/generic/webFeed/templates/rss2.tpl	2015-08-28 18:09:59.889889126 +0200
+++ b/plugins/generic/webFeed/templates/rss2.tpl	2015-09-12 00:17:30.836454681 +0200
@@ -62,6 +62,11 @@
 
 					{* optional elements *}
 					<author>{$article->getAuthorString()|escape:"html"}</author>
+
+                    {foreach from=$article->getGalleys() item=galley}
+
+                                        <dai-file>{url page="article" op="download" path=$article->getBestArticleId($currentJournal)}{"/"}{$galley->getId()}</dai-file>
+                    {/foreach}{* galleys *}
 					{* <category/> *}
 					{* <comments/> *}
 					{* <source/> *}
